--- Atmosphere.cpp.before
+++ Atmosphere.cpp.after
@@ -4,7 +4,7 @@
 using namespace TEN::Scripting;
 using namespace TEN::Scripting::Types;
 
-/** Rain weather settings. To be used with @{Flow.WeatherProfile.rain}.
+/* Rain weather settings. To be used with @{Flow.WeatherProfile.rain}.
 @tenprimitive Flow.RainProfile
 @pragma nostrip
 */
@@ -41,7 +41,7 @@
 	);
 }
 
-/** Weather profile settings. To be used with @{Flow.Atmosphere.weather}.
+/* Weather profile settings. To be used with @{Flow.Atmosphere.weather}.
 @tenprimitive Flow.WeatherProfile
 @pragma nostrip
 */
@@ -74,7 +74,7 @@
 	);
 }
 
-/** Wind profile settings. To be used with @{Flow.Atmosphere.wind}.
+/* Wind profile settings. To be used with @{Flow.Atmosphere.wind}.
 @tenprimitive Flow.WindProfile
 @pragma nostrip
 */
@@ -111,7 +111,7 @@
 	);
 }
 
-/** Aurora sky effect settings. To be used with @{Flow.Atmosphere.aurora}.
+/* Aurora sky effect settings. To be used with @{Flow.Atmosphere.aurora}.
 @tenprimitive Flow.AuroraProfile
 @pragma nostrip
 */
@@ -172,7 +172,7 @@
 	);
 }
 
-/** Moon sky object settings. To be used with @{Flow.Atmosphere.moon}.
+/* Moon sky object settings. To be used with @{Flow.Atmosphere.moon}.
 @tenprimitive Flow.MoonProfile
 @pragma nostrip
 */
@@ -237,7 +237,7 @@
 	);
 }
 
-/** Data for one generated or anchored atmosphere effect layer. To be used with @{Flow.Atmosphere.effects}.
+/* Data for one generated or anchored atmosphere effect layer. To be used with @{Flow.Atmosphere.effects}.
 @tenprimitive Flow.AtmosphereEffectProfile
 @pragma nostrip
 */
@@ -374,7 +374,7 @@
 	);
 }
 
-/** Data for one global or anchored light shaft. To be used with @{Flow.Atmosphere.lightShafts}.
+/* Data for one global or anchored light shaft. To be used with @{Flow.Atmosphere.lightShafts}.
 @tenprimitive Flow.LightShaftProfile
 @pragma nostrip
 */
@@ -451,7 +451,7 @@
 	);
 }
 
-/** Atmosphere settings. To be used with @{Flow.Level.atmosphere}.
+/* Atmosphere settings. To be used with @{Flow.Level.atmosphere}.
 @tenprimitive Flow.Atmosphere
 @pragma nostrip
 */
